.github/ISSUE_TEMPLATE/config.yml

```
@@ -3,9 +3,9 @@ contact_links:
   - name: Support Request
     url: https://sentry.io/support
     about: Use our dedicated support channel for paid accounts.
-  - name: Ask a question about self-hosting/on-premise
-    url: https://forum.sentry.io
-    about: Please use the community forums for questions about self-hosting.
+  - name: Ask a question about self-hosted Sentry
+    url: https://github.com/getsentry/self-hosted/issues/new/choose
```

Nit-picking :) In case some day github changed this url.
```suggestion
    url: https://github.com/getsentry/self-hosted/issues/new
```

---

I just confirmed from a test GitHub account that https://github.com/getsentry/self-hosted/issues/new will redirect to `./choose` for outsiders to the project (not sure of the exact permissions). Given that (we want to force outsiders to use our issue templates), I'm good with this. 👍